saleor/graphql/account/i18n.py

```diff
@@ -11,7 +12,16 @@ class I18nMixin:
     """
 
     @classmethod
-    def validate_address(cls, address_data, instance=None):
+    def validate_address(cls, address_data: dict, instance=None):
+        phone = address_data.get("phone", None)
+        if phone:
+            value = phonenumber.to_python(phone, address_data.get("country"))
```

Comment:
I think there should be one function to validate phone numbers and we already have `validate_possible_number` in `saleor/account/validators.py` which seems pretty similar.